CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 This release uses Write-Ahead Logging (WAL) for the remote_write API. This currently causes a slight increase in memory usage, which will be addressed in future releases.
 
+* [CHANGE] Default time retention is used only when no size based retention is specified. #5216
```

Comment:
I'd mention that this is a flag, and mention the flag names. It'll make it easier for users to realise what's going on.

Reply:
should we then mention the default value?

Reply:
Default values are from the start, I don't think that I required.